src/Sylius/Bundle/CurrencyBundle/spec/Form/Type/CurrencyTypeSpec.php

```diff
@@ -46,6 +46,16 @@ function it_should_build_form_with_proper_fields(FormBuilder $builder)
         ;
 
         $builder
+            ->add('enabled', 'checkbox', Argument::any())
+            ->shouldBeCalled()->willReturn($builder)
+        ;
+
+        $builder
+            ->add('base', 'checkbox', Argument::any())
```

Comment:
And as @pjedrzejewski mention [here](https://github.com/Sylius/Sylius/pull/4473/files#r56580626) it should be removed as well